test/conftest.py

```diff
@@ -0,0 +1,30 @@
+import pytest
+
+import psqlgraph
+
+
+@pytest.fixture(scope="session")
+def pg_conf():
+    return {
+        'host': "localhost",
+        'user': "test",
+        'password': "test",
+        'database': "automated_test"
+    }
```

Comment:
we moved away from this in the API, maybe we just put it as a dictionary into `pg_driver` here as well?